### PR TITLE
LogPrinter: Change log type DEBUG to log type info

### PR DESCRIPTION
**Pull request by mgracer48**
Instead of asking the user to run coala with -L DEBUG It's  better to have exception tracebacks default

Closes https://github.com/coala/coala/issues/2660

Warning: WIP
Have to fix this bug

### CODE REVIEW

**Comment by Nosferatul** [comment]
Hello there! You should have a newline between shortlog _LogPrinter: Change log type DEBUG to log type info_ and the body _Instead of asking the user to run coala with -L DEBUG It's  better to have exception tracebacks default_ . Also you should check [https://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html](url)

**Comment by Nosferatul** [comment]
Also you will need to do a rebase after you pass the checks!

File: tests/output/printers/LogPrinterTest.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def test_logging(self):
         uut.log_message(self.log_message, end="")
         self.assertEqual(uut.printer.string, str(self.log_message))
 
-        uut = LogPrinter(StringPrinter(), log_level=LOG_LEVEL.DEBUG)
+        uut = LogPrinter(StringPrinter(), log_level=LOG_LEVEL.INFO)
```
Comment (Makman2) [inline tests/output/printers/LogPrinterTest.py]: This is the cause of the failing test! :D
The `log_level` parameter sets the minimum log-level to print. `DEBUG` is the lowest one, `INFO` follows. So please don't change this line, then tests work fine :D

File: coalib/output/printers/LogPrinter.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def log_exception(self,
 
         self.log(log_level, message, timestamp=timestamp, **kwargs)
```
Comment (Adrianzatreanu) [inline coalib/output/printers/LogPrinter.py]: are there 2 spaces between "It's" and "better"? make it one please.

also, it should be "It's"->"it's".

needs comma after the "-L DEBUG"

enclose "-L DEBUG" in double backticks as its a command.

end body with period.

**Pull request closed without merging**